charts/refinery/CHANGELOG.md

```diff
@@ -1,5 +1,9 @@
 # Refinery Helm Chart Changelog
 
+## Refinery v1.19.1
+  - feat: Add the ability to create your own Secret Provider (#243) | @RainofTerra
```

Comment:
It doesn't look like we've done it the last couple of times, but in the past and in other repositories, we use a link to the user instead of just the @. Example: `[@TylerHelmuth](https://github.com/TylerHelmuth)`

Reply:
Ya I've been purposefully updating the user links bc the `@`s in the release notes auto-generate links and more importantly auto-generate the Contributors section.